### PR TITLE
OWLS-91272 - Add notes on using patched CPU image in production to documentation and create domain input yaml files

### PR DESCRIPTION
**Pull request by alai8**

### CODE REVIEW

File: kubernetes/samples/quick-start/domain-resource.yaml [hunk 1]
```diff
@@ -42,6 +42,13 @@ spec:
   domainHome: /u01/domains/sample-domain1
 
   # The WebLogic Server image that the Operator uses to start the domain
+  # **NOTE**:
+  # This example uses General Availability (GA) images. GA images are suitable for demonstration and
+  # development purposes only where the environments are not available from the public Internet;
+  # they are not acceptable for production use. In production, you should always use CPU (patched)
+  # images from OCR or create your images using the WebLogic Image Tool.
+  # Please refer to the `OCR` and `Weblogic Images` pages in the WebLogic Kubernetes Operator
```
Comment (rosemarymarano) [inline kubernetes/samples/quick-start/domain-resource.yaml:50]: Weblogic Images -> WebLogic Images

File: kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml [hunk 2]
```diff
@@ -150,9 +150,14 @@ domainPVMountPath: /shared
...
-# Refer to [WebLogic Server images](../../../../../site/weblogic-server-images.md) for details on how
-# to obtain or create the image.
-# See README.md for more help.
+#
+# **NOTE**:
+# This sample uses General Availability (GA) images. GA images are suitable for demonstration and
+# development purposes only where the environments are not available from the public Internet;
+# they are not acceptable for production use. In production, you should always use CPU (patched)
+# images from OCR or create your images using the WebLogic Image Tool.
+# Please refer to the `OCR` and `Weblogic Images` pages in the WebLogic Kubernetes Operator
```
Comment (rosemarymarano) [inline kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml:159]: Weblogic Images -> WebLogic Images

File: kubernetes/samples/scripts/create-weblogic-domain-on-azure-kubernetes-service/create-domain-on-aks-inputs.yaml [hunk 3]
```diff
@@ -83,6 +83,13 @@ weblogicAccountPassword: weblogic-account-password
 
 # WebLogic Server image.
 # Parameter "image" will be overwritten with this field in kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/create-domain-inputs.yaml
+# **NOTE**:
+# This sample uses General Availability (GA) images. GA images are suitable for demonstration and
+# development purposes only where the environments are not available from the public Internet;
+# they are not acceptable for production use. In production, you should always use CPU (patched)
+# images from OCR or create your images using the WebLogic Image Tool.
+# Please refer to the `OCR` and `Weblogic Images` pages in the WebLogic Kubernetes Operator
```
Comment (rosemarymarano) [inline kubernetes/samples/scripts/create-weblogic-domain-on-azure-kubernetes-service/create-domain-on-aks-inputs.yaml:91]: Weblogic Images -> WebLogic Images

File: kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/create-domain-inputs.yaml [hunk 4]
```diff
@@ -40,8 +40,14 @@ managedServerPort: 8001
...
 # The WebLogic Server image, `weblogic:12.2.1.3`, has all the necessary patches applied.
-# Refer to [WebLogic Server images](../../../../../site/weblogic-server-images.md) for details on how
-# to obtain or create the image.
+#
+# **NOTE**:
+# This sample uses General Availability (GA) images. GA images are suitable for demonstration and
+# development purposes only where the environments are not available from the public Internet;
+# they are not acceptable for production use. In production, you should always use CPU (patched)
+# images from OCR or create your images using the WebLogic Image Tool.
+# Please refer to the `OCR` and `Weblogic Images` pages in the WebLogic Kubernetes Operator
```
Comment (rosemarymarano) [inline kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/create-domain-inputs.yaml:49]: Weblogic Images -> WebLogic Images

File: kubernetes/samples/scripts/create-weblogic-domain/model-in-image/domain-resources/WLS-AI/mii-initial-d1-WLS-AI-v1.yaml [hunk 5]
```diff
@@ -20,6 +20,13 @@ spec:
   domainHome: /u01/domains/sample-domain1
 
   # The WebLogic Server image that the Operator uses to start the domain
+  # **NOTE**:
+  # This sample uses General Availability (GA) images. GA images are suitable for demonstration and
+  # development purposes only where the environments are not available from the public Internet;
+  # they are not acceptable for production use. In production, you should always use CPU (patched)
+  # images from OCR or create your images using the WebLogic Image Tool.
+  # Please refer to the `OCR` and `Weblogic Images` pages in the WebLogic Kubernetes Operator
```
Comment (rosemarymarano) [inline kubernetes/samples/scripts/create-weblogic-domain/model-in-image/domain-resources/WLS-AI/mii-initial-d1-WLS-AI-v1.yaml:28]: Weblogic Images -> WebLogic Images

File: kubernetes/samples/scripts/create-weblogic-domain/model-in-image/domain-resources/WLS-AI/mii-update1-d1-WLS-AI-v1-ds.yaml [hunk 6]
```diff
@@ -20,6 +20,13 @@ spec:
   domainHome: /u01/domains/sample-domain1
 
   # The WebLogic Server image that the Operator uses to start the domain
+  # **NOTE**:
+  # This sample uses General Availability (GA) images. GA images are suitable for demonstration and
+  # development purposes only where the environments are not available from the public Internet;
+  # they are not acceptable for production use. In production, you should always use CPU (patched)
+  # images from OCR or create your images using the WebLogic Image Tool.
+  # Please refer to the `OCR` and `Weblogic Images` pages in the WebLogic Kubernetes Operator
```
Comment (rosemarymarano) [inline kubernetes/samples/scripts/create-weblogic-domain/model-in-image/domain-resources/WLS-AI/mii-update1-d1-WLS-AI-v1-ds.yaml:28]: Weblogic Images -> WebLogic Images

File: kubernetes/samples/scripts/create-weblogic-domain/model-in-image/domain-resources/WLS-AI/mii-update2-d2-WLS-AI-v1-ds.yaml [hunk 7]
```diff
@@ -20,6 +20,13 @@ spec:
   domainHome: /u01/domains/sample-domain2
 
   # The WebLogic Server image that the Operator uses to start the domain
+  # **NOTE**:
+  # This sample uses General Availability (GA) images. GA images are suitable for demonstration and
+  # development purposes only where the environments are not available from the public Internet;
+  # they are not acceptable for production use. In production, you should always use CPU (patched)
+  # images from OCR or create your images using the WebLogic Image Tool.
+  # Please refer to the `OCR` and `Weblogic Images` pages in the WebLogic Kubernetes Operator
```
Comment (rosemarymarano) [inline kubernetes/samples/scripts/create-weblogic-domain/model-in-image/domain-resources/WLS-AI/mii-update2-d2-WLS-AI-v1-ds.yaml:28]: Weblogic Images -> WebLogic Images

File: kubernetes/samples/scripts/create-weblogic-domain/model-in-image/domain-resources/WLS-AI/mii-update3-d1-WLS-AI-v2-ds.yaml [hunk 8]
```diff
@@ -20,6 +20,13 @@ spec:
   domainHome: /u01/domains/sample-domain1
 
   # The WebLogic Server image that the Operator uses to start the domain
+  # **NOTE**:
+  # This sample uses General Availability (GA) images. GA images are suitable for demonstration and
+  # development purposes only where the environments are not available from the public Internet;
+  # they are not acceptable for production use. In production, you should always use CPU (patched)
+  # images from OCR or create your images using the WebLogic Image Tool.
+  # Please refer to the `OCR` and `Weblogic Images` pages in the WebLogic Kubernetes Operator
```
Comment (rosemarymarano) [inline kubernetes/samples/scripts/create-weblogic-domain/model-in-image/domain-resources/WLS-AI/mii-update3-d1-WLS-AI-v2-ds.yaml:28]: Weblogic Images -> WebLogic Images

**Review by rosemarymarano — COMMENTED** [review]
One edit, needed in a few places.

File: kubernetes/samples/quick-start/domain-resource.yaml (hunk 1 above)
Comment (alai8) [reply kubernetes/samples/quick-start/domain-resource.yaml]: Fixed in all places. Thanks!

**Review by rosemarymarano — APPROVED** [review]

**Review by rjeberhard — APPROVED** [review]

**Pull request merged**